yt/data_objects/profiles.py

```diff
@@ -1347,7 +1347,14 @@ def create_profile(
         else:
             logs_list.append(data_source.ds.field_info[bin_field].take_log)
     logs = logs_list
-    if extrema is None:
+
+    # Are the extrema all Nones? Then treat them as though extrema was set as None
+    if extrema is not None:
+        extrema_vals = list(extrema.values())
+        flat_list = [item for sublist in extrema_vals for item in sublist]
+        extrema_all_nones = not any(flat_list)
+
+    if extrema is None or extrema_all_nones:
```

Comment:
This can be simplifies as
```suggestion
    if extrema is None or not any(collapse(extrema.values())):
```
(this will also requiring adding an import at the top level: `from more_itertools import collapse`)